src/common/src/util/resource_util.rs

```diff
@@ -260,6 +271,16 @@ pub mod cpu {
 
     // Returns the CPU limit when cgroup_V1 is utilised.
     fn get_cpu_limit_v1() -> Result<f32, std::io::Error> {
+        let is_max = super::util::is_max_in_file_path(&format!(
+            "{}{}",
+            super::DEFAULT_CGROUP_ROOT_HIERARCYHY,
+            V1_CPU_QUOTA_HIERARCHY
+        ))?;
+        if is_max {
+            let max_cpu = thread::available_parallelism()?;
+            return Ok(max_cpu.get() as f32);
+        }
+
```

Comment:
I believe the path is a fixed at runtime so please use a variable to store and reuse instead of computing it once again.